### PR TITLE
Remove Github OAuth

### PR DESCRIPTION
**Pull request by cadecairos**

### CODE REVIEW

**Comment by cadecairos** [comment]
@jbuck R?

**Comment by jbuck** [comment]
@cadecairos travis fails

**Comment by cadecairos** [comment]
@jbuck travis is green!

File: server/models/github.js [hunk 1]
```diff
@@ -12,12 +12,10 @@ var request = require('request');
...
  * Constructor
  */
-function Github(githubSecrets, cacheAge) {
+function Github(token, cacheAge) {
   var _this = this;
-
-  _this.client = githubSecrets.client;
-  _this.secret = githubSecrets.secret;
-  _this.token = githubSecrets.token;
+console.log(token);
```
Comment (jbuck) [inline server/models/github.js]: Probably safe to delete this yeah?

**Comment by jbuck** [comment]
@cadecairos r+ with one nit noted in the PR

File: server/models/github.js (hunk 1 above)
Comment (cadecairos) [inline server/models/github.js]: noooooo

**Pull request merged**